### ISSUE TITLE
Physics don't change in splitscreen mode

### ISSUE THREAD

**Issue by Spiderfffun** [bmx22c/tm-physics-changer#1]
Title is self explainatory, clip triggers but the physics dont change

**Comment by bmx22c** [comment]
Does it even trigger for one of the players ?

**Comment by Spiderfffun** [comment]
> Does it even trigger for one of the players ?

No, i added car trails and the clip activates but it doesn't work. (I did test this only with acceleration tho, but will post another comment testing everything.)

**Comment by Spiderfffun** [comment]
Tested with trails: Trails activate on both players, physics don't activate at all (having everything set to 0.5)
